build.sbt

```diff
@@ -103,6 +103,7 @@ lazy val publishSettings : Seq[Setting[_]] = Seq(
     if (file.exists) List(Credentials(file))
     else Nil
   },
+  ivyConfigurations += Configuration("default", "Default", true, List(Configurations.Runtime), true),
```

Comment:
could you add a comment to this line? it's not obvious for an occasional visitor.